tests/zoomus/components/webinar/test_add_panelists.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+import unittest
+
+from zoomus import components
+import responses
+
+
+def suite():
+    """Define all the tests of the module."""
+    suite = unittest.TestSuite()
+    suite.addTest(unittest.makeSuite(AddPanelistsV2TestCase))
+    return suite
+
+
+class AddPanelistsV2TestCase(unittest.TestCase):
+    def setUp(self):
+        self.component = components.webinar.WebinarComponentV2(
+            base_uri="http://foo.com",
+            config={
+                "api_key": "KEY",
+                "api_secret": "SECRET",
+                "version": util.API_VERSION_2,
+            },
+        )
+
+   @responses.activate
+    def test_can_add_panelists(self):
+        responses.add(
+            responses.POST, "http://foo.com/webinar/ID/panelists",
+        )
+        response = self.component.add_panelists(panelists=[{"name": "Mary", "email": "maryjkdfdsgfshdgf@jdfdkjdglfk.jkfgdj"}])
```

Comment:
It's perfectly fine to have a fake email, but can we make it something like 'test@test.com' just to make it look a bit cleaner? Thanks!

Reply:
done.